control/xferfcn.py

```diff
@@ -1349,3 +1317,42 @@ def tfdata(sys):
     tf = _convertToTransferFunction(sys)
 
     return (tf.num, tf.den)
+
+def _cleanPart(data):
+    '''
+    Return a valid, cleaned up numerator or denominator 
+    for the TransferFunction class.
+    
+    Parameters:
+    data: numerator or denominator of a transfer function.
+    
+    Returns:
+    data: correctly formatted transfer function part.
```

Comment:
you could be a little more specific: return data as a list of lists of ndarrays of type float.